package/upgrade/upgrade_node.sh

```diff
@@ -508,6 +508,55 @@ calculateCPUReservedInMilliCPU() {
   echo $reserved
 }
 
+replace_with_mgmtvlan() {
+  #files to update
+  SETUPBOND_FILE="${HOST_DIR}/etc/wicked/scripts/setup_bond.sh"
+  SETUPBRIDGE_FILE="${HOST_DIR}/etc/wicked/scripts/setup_bridge.sh"
+  CUSTOM90_FILE="${HOST_DIR}/oem/90_custom.yaml"
+  CONFIG_FILE="${HOST_DIR}/oem/harvester.config"
+
+  # Check if the harvester config file exists
+  if [[ ! -f "$CONFIG_FILE" ]]; then
```

Comment:
return 1 will cause the ugprade to fail, and will always fail when the pod re-run

log an error/warning, then continue or return